FLIR/conservator/local_dataset.py

```diff
@@ -766,7 +766,10 @@ def _download_and_link(self, asset, max_retries=5):
                 no_meter=True,
                 max_retries=max_retries,
             )
-            LocalDataset._add_links(download_path, paths_to_link, use_symlink)
+            try:
```

Comment:
The `try/except/raise` code added in this module is pointless. Please remove it.